content/docs/configuration/acme/dns01/route53.md

```diff
@@ -47,30 +47,382 @@ permissions:
+   ```
+
+### Non-ambient Credentials
+
+Non-ambient credentials are credentials which are explicitly configured on the Issuer or ClusterIssuer resource.
+For example:
+- **Access key Secret reference**:<br/>
+  where cert-manager loads a long-term access key from a Kubernetes Secret resource.
+- **ServiceAccount reference**:<br/>
+  where cert-manager gets a ServiceAccount token (signed JWT) from the Kubernetes API server,
+  and uses the STS AssumeRoleWithWebIdentity endpoint to exchange it for temporary AWS credentials.
+
+The advantage of non-ambient credentials is that cert-manager can perform Route53 operations in a multi-tenant environment.
+Each tenant can be granted permission to create and update Issuer resources in their namespace and they can provide their own AWS credentials in their namespace.
+
+#### Referencing your own ServiceAccount within in an Issuer or ClusterIssuer
+
+> 📖 Read the [AWS + LoadBalancer + Let's Encrypt tutorial](../../../tutorials/getting-started-aws-letsencrypt/README.md)
+> to learn how to deploy cert-manager on EKS and use this authentication mechanism.
+
+In this configuration you can reference your own `ServiceAccounts` in your `Issuer` or `ClusterIssuer`
+and cert-manager will get a ServiceAccount token from the Kubernetes API which it will send to STS in exchange for AWS temporary credentials.
+The advantage of this method over IRSA or Pod Identity is that each Issuer can reference a different `ServiceAccount`,
+which means you can lock down the permissions,
+such that each `ServiceAccount` is mapped to an IAM role that only has permission to update the zones it needs for that particular
+
+1. **Create a ServiceAccount**
+
+   In order to reference a `ServiceAccount` it must first exist.
+   Unlike normal IRSA the `eks.amazonaws.com/role-arn` annotation is not required.
+
+   ```yaml
+   apiVersion: v1
+   kind: ServiceAccount
+   metadata:
+     name: <service-account-name>
+   ```
+
+2. **Create an IAM role trust policy**
+
+   For every `ServiceAccount` you want to use for AWS authentication you must first set up a trust policy:
+
+   ```json
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Action": "sts:AssumeRoleWithWebIdentity",
+         "Principal": {
+           "Federated": "arn:aws:iam::<aws-account-id>:oidc-provider/oidc.eks.<aws-region>.amazonaws.com/id/<eks-hash>"
+         },
+         "Condition": {
+           "StringEquals": {
+             "oidc.eks.<aws-region>.amazonaws.com/id/<eks-hash>:sub": "system:serviceaccount:<namespace>:<service-account-name>"
+           }
+         }
+       }
+     ]
+   }
+   ```
+
+   Replace the following:
+
+   - `<aws-account-id>` with the AWS account ID of the EKS cluster.
+   - `<aws-region>` with the region where the EKS cluster is located.
+   - `<eks-hash>` with the hash in the EKS API URL; this will be a random 32 character hex string (example: `45DABD88EEE3A227AF0FA468BE4EF0B5`)
```

Comment:
nitpick (non-blocking): Really unimportant but all other bullets in the list end in a period and this one doesn't!